onboarding/mlsys/1-gradients.md

```diff
@@ -491,3 +512,14 @@ reduces network communications.  If you always aggregate gradients from 4
 batches before applying them to the model, you only need to communicate
 gradients between workers 1/4th of the time, significantly reducing your
 communication overhead.
+
+## Summary
+
+Upon completion of this exercise, you have learned:
+
+- the hairy details of gradient descent
+- that averaging gradients allows for faster training
+- how batching data is just a form of averaging gradients
+- how data-parallel dtrain is just a different form of averaging gradients
+- why gradient aggregation (a third form of averaging gradients) is
+  complementary to data-parallel dtrain
```

Comment:
Since we capitalize most everything else, we should probably also capitalize these.